lib/render_async/view_helper.rb

```diff
@@ -4,6 +4,9 @@ module RenderAsync
   module ViewHelper
 
     def render_async(path)
+      cached_view = Rails.cache.read("views/#{path}_cached")
```

Comment:
What if we are also passing different parameters? Shouldn't we cache based on not just path but params as well?